docs/decisions/0008-frontend-plugin-fallback.rst

```diff
@@ -0,0 +1,38 @@
+How to Handle Frontend iFrame Plugin Fallback 
+======================================================================================
+
+Status
+------
+
+Draft
+
+Context
+-------
+The Frontend Plugin Framework's iFrame class implementation will live in Frontend Platform,
+but the question of how we want to handle the case where the iFrame fails to render needs to
+handled. 
+
+Given that this is a new feature that is being made available to the frontend, we 
+can't anticipate all of the ways that the plugin will be used for a given repo, nor do we 
+have any intention to limit the scope of how it can be used. Because we don't all of the use 
+cases for the plugin, we also don't know the desired fallback method that is desired for any 
+given plugin.
+
+Decision
+--------
+For the above reasons, we will have failing iFrame plugins return an H1 tag that 
+says "There was an error.". By doing so, it is up to the engineer/team that implements the plugin 
+to handle the desired fallback however they would like. 
```

Comment:
How will we do that? By checking for the element returned by the pluginSlot?

Reply:
How we will be able to check the element returned by the pluginSlot?

Reply:
hey all! huge apologies for the delay on these responses. After having written this ADR draft, I realized that the PR I was taking on wasn't in any state for me to confidently know _what worked_ and thus a lack of confidence in the ways we could handle the fallback. 

I will be editing this draft to reflect my findings and we'll ideally have a clearer picture to work off of.